modules/gapi/perf/common/gapi_core_perf_tests_inl.hpp

```diff
@@ -2196,9 +2379,10 @@ PERF_TEST_P_(ResizeFxFyPerfTest, TestPerformance)
 
 PERF_TEST_P_(BottleneckKernelsConstInputPerfTest, TestPerformance)
 {
-    compare_f cmpF = get<0>(GetParam());
-    std::string fileName = get<1>(GetParam());
-    cv::GCompileArgs compile_args = get<2>(GetParam());
+    compare_f cmpF;
+    std::string fileName = "";
+    cv::GCompileArgs compile_args;
+    std::tie(cmpF, fileName, compile_args) = GetParam();
```

Comment:
Could you please create `double fx, fy` variables initialize them with `0.5` value and then replace `0.5` value in functions calls with fx, fy variables?

Reply:
Please do it for `ResizeInSimpleGraph` test too.

Reply:
Applied